Jenkinsfile

```diff
@@ -95,6 +111,20 @@ parallel(
 					}
 
 					sh 'make bootstrap'
+
+					if (SDK_GO_REPO && SDK_SHA) {
+						echo "Updating local glide.yaml with controller-sdk-go repo '${SDK_GO_REPO}' and version '${SDK_SHA}'"
+
+						def pattern = "github\\.com\\/deis\\/controller-sdk-go\\n\\s+version:\\s+[a-f0-9]+"
+						def replacement = "${SDK_GO_REPO.replace("/", "\\/")}\\n  version: ${SDK_SHA}"
+						sh "perl -i -0pe 's/${pattern}/${replacement}/' glide.yaml"
+
+						def glideYaml = readFile('glide.yaml')
+						echo "Updated glide.yaml:\n${glideYaml}"
+
+						sh 'make glideup'
```

Comment:
we are updating the controller-sdk-go package's version number, which then glide.lock is out of date so `glide install` won't work in this case.